doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -129,6 +129,81 @@ This is a reference configuration that can be modified in the production firmwar
        To use the Oberon backend for specific cryptographic operations supported by both drivers, disable those operations in the CRACEN driver, as it takes priority when both are enabled.
        See the :ref:`nrf_security_drivers` documentation for more information.
 
+.. _matter_platforms_security_dac_priv_key:
+
+Storing Device Attestation Certificate private key
+**************************************************
+
+In Matter samples based on the PSA crypto API, the Device Attestation Certificate's private key, which exists in the factory data set, can be migrated to secure storage.
+The secure storage used depends on the platform and the cryptographic backend.
+
+To enable the migration of the DAC private key from the factory data set to secure storage, set the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY` Kconfig option to ``y``.
+
+Currently, this feature is available only for the PSA crypto API.
+See the following table to learn about the default secure storage backends for the DAC private key and the available secure storage backends for each platform:
+
+.. list-table:: Matter secure storage
+   :widths: auto
+   :header-rows: 1
+
+   * - Platform
+     - Default secure storage backend for DAC private key
+     - Available secure storage backends
+   * - nRF52840 SoC
+     - Trusted Storage library + SHA-256 hash
+     - Trusted Storage library + SHA-256 hash
+   * - nRF5340 SoC
+     - Thread
+     - Trusted Storage library + Hardware Unique Key (HUK), Trusted Storage library + SHA-256 hash
+   * - nRF5340 SoC + nRF7002 companion IC
+     - Not available
+     - Not available
+   * - nRF54L15 SoC
+     - Trusted Storage library + Hardware Unique Key (HUK)
+     - Key Management Unit (KMU), Trusted Storage library + Hardware Unique Key (HUK),  Trusted Storage library + SHA-256 hash
+   * - nRF54L15 SoC + Trusted Firmware-M (TF-M)
+     - Trusted Storage library + Hardware Unique Key (HUK)
+     - Key Management Unit (KMU), Trusted Firmware-M (TF-M) Storage
+
+.. _matter_platforms_security_dac_priv_key_its:
+
+DAC in Trusted Storage library
+==============================
+
+The Device Attestation Certificates private key can be stored in the Trusted Storage library.
+The key is encrypted with the AEAD key derived from the Hardware Unique Key (HUK) or a SHA-256 hash.
+This storage backend is selected by default for all platforms that support the PSA crypto API.
+
+To enable storing the DAC private key in the Trusted Storage library, set the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_ITS` Kconfig option to ``y``.
+To select encryption choose one of the possible options:
+
+- To use key derivation from HUK, set the :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK` Kconfig option to ``y``.
+- To use key derivation from a SHA-256 hash, set the :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_HASH_UID` Kconfig option to ``y``.
```

Comment:
```suggestion
- To use key derivation from HUK, set :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK` to ``y``.
- To use key derivation from a SHA-256 hash, set :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_HASH_UID` to ``y``.
```
Can leave out the Kconfig part here since we added it to the introducing sentence.